docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,246 @@
+# Institutional FAQ
+
+This page contains common questions from users of JupyterHub,
+broken down by their roles within organizations.
+
+# For all
+
+## Is appropriate for adoption within a larger institutional context?
+
+Yes! JupyterHub has been used at-scale for large pools of users, as well
+as complex and high-performance computing. For example, UC Berkeley uses
+JupyterHub for its Data Science Education Program courses (serving over
+3,000 students). The Pangeo project uses JupyterHub to provide access
+to scalable cloud computing with Dask. JupyterHub is stable customizable
+to the use-cases of large organizations.
+
+## I keep hearing about Jupyter Notebook, JupyterLab, and now JupyterHub. What’s the difference?
+
+Here is a quick breakdown of these three tools:
+
+* **The Jupyter Notebook** is a document specification (the `.ipynb`) file that interweaves
+  narrative text with code cells and their outputs. It is also a graphical interface
+  that allows users to edit these documents
+* **JupyterLab** is a flexible and extendible user interface for interactive computing. It
+  has several extensions that are tailored for using Jupyter Notebooks, as well as extensions
+  for other parts of the data science stack.
+* **JupyterHub** is an application that can manage **multiple users** with interactive computing
```

Comment:
How about "... is an application that manages interactive computing sessions for **multiple users**. As well as connect them with infra..."?